tests/test_api/test_endpoints.py

```diff
@@ -262,8 +262,7 @@ def test_submit_workflow_activities(api_site_client):
                     "nmdc:dobj-11-g6d71n77",
                     "nmdc:dobj-11-bds7qq03",
                 ],
-                "type": "nmdc:ReadQcAnalysisActivity",
-                "part_of": ["nmdc:omprc-11-9mvz7z22"],
+                "type": "nmdc:ReadQcAnalysis",
```

Comment:
do we want to also ensure that the `associated_studies` value is correct?

Reply:
associated_studies is not a valid slot on this class and this is a dummy record, but was_informed_value does exist with a corresponding nmdc:wfrqc-11-t0tvnp52.1 record. Just FYI If this data was reanalyzed this test would fail as there would already be a nmdc:wfrqc-11-t0tvnp52.2 record in the system.

Reply:
As the test is currently constructed, we will first drop any documents in the test collection where the id matches the id that the minter should create (eg nmdc:wfrqc-11-t0tvnp52.2) in above.